app/helpers/application_helper.rb

```diff
@@ -23,6 +23,11 @@ def home_page?
     params.values_at(:controller, :action) == %w[pages index]
   end
 
+  def open_petition_page?
+    # XXX What should go here? Same as petition_page? plus petition.open? Is that allowed here?
+    true
```

Comment:
The method `petition_page?` is just a check for the controller and action name but we know that if it's true then the `@petition` instance variable is set so just use the `Petition#open?` method to achieve what you need, e.g:

```
petition_page? && @petition.open?
```

Hopefully that's clearer?